.env

```diff
@@ -0,0 +1,10 @@
+# Before running command `npm run deploy`
```

Comment:
`.env` is not versioned, you should create a `.env.example` and let the users create their own `.env`. This is common practice.